### DISCUSSION TITLE
Add digital certificate to  Excel file

### DISCUSSION THREAD

**Discussion by dittybijil** [exceljs/exceljs#2537] [Ideas & Proposals]
Is it possible to implement the feature of adding a digital certificate to an Excel file?

**Comment by zurmokeeper** [comment]
@dittybijil Can you describe this function in detail? Or some examples in office excel?

**Comment by dittybijil** [comment]
@zurmokeeper 
It appears that https://github.com/exceljs/exceljs/issues/1710 ticket may also pertain to a similar matter. For further information, kindly refer to the relevant details available at the following link: 

https://docs.microsoft.com/en-us/office/troubleshoot/excel/digital-signatures-code-signing.

**Comment by samuraitruong** [comment]
I raised #1710 few year ago when work on  a banking solution need Digital Signature support. I ended up with using 3rd library in .NET to write API service and sign the excel file  after generate with exceljs.

To support digital signature it will requires lots of dev works. If exceljs can support that will be great

**Comment by Siemienik** [comment]
If still someone need this, please contact me on exceljs@siemienik.com - I will to estimate cost of doing that.
